app/utilities/firebase/firebase.tsx

```diff
@@ -0,0 +1,7 @@
+import firebase from "firebase"
+import firebaseConfig from "../../env/firebaseConfig.json"
+
+  // Initialize Firebase
+  firebase.initializeApp(firebaseConfig);
```

Comment:
`firebase.apps.length ? firebase.app() : firebase.initializeApp(firebaseConfig)`

Reply:
what does this do?

Reply:
just a safety precaution?

Reply:
yeah I think I've seen this a lot more. I think it just make sure we don't create app twice